build.yaml

```diff
@@ -1798,9 +1798,9 @@ steps:
      set +e
      gcloud -q compute instances list \
          --filter 'tags.items=batch2-agent AND labels.namespace={{ default_ns.name }}' \
-         --format="value(name)" \
+         --format="table[no-heading](zone.basename(), name)" \
          --project {{ global.project }} \
-       | xargs -r gcloud -q compute instances delete --zone {{ global.zone }} --project {{ global.project }} || true
+       | xargs -n2 -r sh -c 'gcloud -q compute instances delete --zone "$1" --project {{ global.project }} "$2" || true' argv0
```

Comment:
well this is some elite bash-ing. I had to verify that `argv0` was doing what I thought it was doing.